data_visualization_in_open_source_software/data_visualization.md

```diff
@@ -0,0 +1,265 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  0.0.1
+language: en
+narrator: UK English Female
+title: Data Visualization in Open Source Software
+comment:  This module introduces ggplot2 and seaborn, popular data visualization libraries in R and python, respectively. It lays the groundwork for using ggplot2 and seaborn by 1) highlighting common features of plots that can be manipulated in plot code and 2) discussing data preparation for plotting.
+
+long_description: This module introduces ggplot2 and seaborn, popular data visualization libraries in R and python, respectively. It lays the groundwork for using ggplot2 and seaborn by 1) highlighting common features of plots that can be manipulated in plot code and 2) discussing data preparation for plotting. This content will be most useful for people who have some experience creating data visualizations and/or reading plots presented in research articles or similar contexts. Some prior exposure to R and/or python is helpful but not required. This is appropriate for beginners.
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+* describe the role ggplot2 and seaborn play in the R and python programming languages, respectively
+* identify key elements in a plot that communicate information about the data
+* discuss possible data manipulation necessary to plot in either seaborn (for python) or ggplot2 (for R)
```

Comment:
hmm I'm not sure this module really accomplishes this objective, at least not primarily. There's a note about the potential need to pivot, but it didn't feel to me like a primary aim of the content. 

Perhaps describing the data viz workflow is a more apt LO? and/or describing the basics of data viz a11y? At the least, it may be a good idea to add both of those as LOs to meet the QA item of "All major topics covered are represented by a learning objective -- there's no significant knowledge imparted that isn't specified in the learning objectives"